src/vector/vector_tests/sql.rs

```diff
@@ -1,14 +1,14 @@
 use std::collections::HashSet;
 
+use crate::test_utils::fixture;
```

Comment:
Nit: we should probably use a consistent style (see line below), but it doesn't matter that much.